tests/functions/test_expand_grid.py

```diff
@@ -181,6 +181,22 @@ def test_Index(df):
     assert_frame_equal(result, expected)
 
 
+@settings(deadline=None)
+@given(df=df_strategy())
+def test_Index_name_none(df):
+    """Test expand_grid output"""
```

Comment:
One small request from me to be a bit more descriptive on what this test does; from the test name, it seems it is specifically testing when value is an Index, with no name?